### PR TITLE
Vs3 set get enable dc w/out PMW

### PR DESCRIPTION
**Pull request by olivierverwoerd**

### CODE REVIEW

File: src/dc_interface.cpp [hunk 1]
```diff
@@ -3,42 +3,25 @@
...
-
-void MotorController::DcInterface::setDirection(const MotorController::MotorInterface::Direction newDirection) {
-    direction = newDirection;
-}
-
 int16_t MotorController::DcInterface::getSpeed() const {
     return speed;
 }
 
 void MotorController::DcInterface::setSpeed(const int16_t newSpeed) {
```
Comment (Druyv) [inline src/dc_interface.cpp:10]: You generally never want to print anything in a function. If you leave out the prints, this bit of logic can be simplified as well.

File: src/main.cpp [hunk 2]
```diff
@@ -115,33 +97,26 @@ int main() {
...
     // TODO: We should see if we can make rtos work, then we can make this function a task
     mainLogic(motorController);
     */
+
     MotorController::Controller motorController;
     constexpr const char question[] = "Select your Interface (0: DC, 1: Stepper)";
     motorController.setSelectedInterface(static_cast<MotorController::Controller::Interface>(askNumber(question)));
     while (1) {
-        // hwlib::cout << "heko";
         motorController.setEnable(true);
```
Comment (Druyv) [inline src/main.cpp]: These functions no longer exist in this version of the document

**Review by Druyv — COMMENTED** [review]
Some brief comments on specific parts of code. On top of that, there is now no longer any functionality (stub or implemented) for turning/changing direction. That used to be there in an older version, where did that go?

**Comment by olivierverwoerd** [comment]
Forward and backward has now been set to positive and negative value as it's more of a standard. Brief doxygen will be fixed later.

File: src/dc_interface.cpp (hunk 1 above)
Comment (olivierverwoerd) [reply src/dc_interface.cpp:10]: ?

Comment (olivierverwoerd) [reply src/dc_interface.cpp:10]: printout is there to help remember the implementation of PWM

**Review by StienoX — CHANGES_REQUESTED** [review]
Please update the doxygen in "controller.hpp", would like to see a negative speed test and negative angle test.

**Review by BendeguzToth — CHANGES_REQUESTED** [review]
Please fix the doxygen of the setter and getter functions.

**Review by BendeguzToth — APPROVED** [review]

**Pull request merged**